pkg/adhoc/writer/external_writer.go

```diff
@@ -97,7 +98,20 @@ func (w *externalWriter) write(name string, out *storage.GetOutput) error {
 			return fmt.Errorf("could not write the collapsed file: %w", err)
 		}
 	case "html":
-		fb := flamebearer.NewProfile(filename, out, w.maxNodesRender)
+		fb := flamebearer.NewProfile(flamebearer.ProfileConfig{
+			Name:      filename,
+			MaxNodes:  w.maxNodesRender,
+			Tree:      out.Tree,
+			Timeline:  out.Timeline,
+			Groups:    out.Groups,
+			Telemetry: out.Telemetry,
```

Comment:
That's funny, none of these seem to be used/relevant when writing externally (ie when exporting). I don't think they should be part of `FlamebearerProfile` struct

Reply:
I agree, `FlamebearerProfile` is super bloated and hard to maintain. We need to revise this type